app-sso/tutorials/app-operators/secure-workload.hbs.md

```diff
@@ -0,0 +1,346 @@
+# Secure a Workload with AppSSO
+
+This topic tells you how to secure a `Workload` running on TAP with
+Application Single Sign-On (commonly called AppSSO).
+
+For specific stack implementations such as _Spring Boot_ and single-page
+applications see the [how-to guides](../../how-to-guides/index.hbs.md).
+
+As we learned in the previous section, we can obtain OAuth2 client credentials
+for an authorization server by claiming them from an AppSSO service offering.
+This is done with the `tanzu service class-claims create` command or with a
+`ClassClaim` resource. In either case, we can customize our OAuth2 client
+settings by providing parameters with our claim. For our `Workload` to be
+secured successfully we have to provide the right parameters for the situation.
+
+The following sections elaborate on each parameter in detail and, eventually, 
+how to load credentials into a `Workload`.
+
+Here's a quick reminder that when iterating on your `ClassClaim` make sure to
+recreate it when you make changes. Updates to an existing `ClassClaim` [have no
+effect](../../../services-toolkit/concepts/class-claim-vs-resource-claim.hbs.md#classclaim)
+
+## OAuth2 client parameters
+
+A `ClassClaim` for an AppSSO service is the request for OAuth2 client
+credentials for an authorization server. Lets revisit the anatomy of a
+`ClassClaim` for an AppSSO service.
+
+The following is an example of claim for a service called `sso` which
+customizes all available client parameters:
+
+```yaml
+---
+apiVersion: services.apps.tanzu.vmware.com/v1alpha1
+kind: ClassClaim
+metadata:
+  name: my-sso-client
+  namespace: my-apps
+spec:
+  classRef:
+    name: sso
+  parameters:
+    workloadRef:
+      name: my-workload
```

Comment:
**[thought]**: we should mention that `my-workload` does not yet have to exist to be referenced here.

Reply:
ack